src/Http/Routing/test/UnitTests/EndpointSelectorContextTest.cs

```diff
@@ -1,9 +1,8 @@
-﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Copyright (c) .NET Foundation. All rights reserved.
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
-using System.Linq;
 using Microsoft.AspNetCore.Http;
-using Microsoft.AspNetCore.Routing.Matching;
+using Microsoft.AspNetCore.Http.Endpoints;
```

Comment:
This Get/SetEndpoint + this namespace are new in 3.0. Not a breaking change if we want to move them

Reply:
Not as part of this PR, that'll affect alot more code than I want to touch